CHANGELOG.md

```diff
@@ -0,0 +1,207 @@
+# Changelog
+
+Darts is still in an early development phase and we cannot always guarantee backwards compatibility. Changes suceptible to **break existing code** working on a previous release are marked with a "&#x1F534;". Sorry for the inconvenience!
+
+## [Unreleased](https://github.com/unit8co/darts/tree/develop)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.3.0...develop)
+
+## [0.3.0](https://github.com/unit8co/darts/tree/0.3.0) (2020-10-dd)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.2.3...0.3.0)
+
+### For users of the library:
+**Added:**
+
+- Indexing on TimeSeries [\#150](https://github.com/unit8co/darts/pull/150)
+- New `FourTheta` forecasting model [\#123](https://github.com/unit8co/darts/pull/123), [\#156](https://github.com/unit8co/darts/pull/156)
+- `map()` method for TimeSeries [\#121](https://github.com/unit8co/darts/issues/121), [\#166](https://github.com/unit8co/darts/pull/166)
+- Further improved Multivariate TimeSeries capabilities:
+  - Added multivariate support to the Neural Network based forecasting models, namely `TorchForecastingModel` and its subclasses (`TCNModel` and `RNNModel`)
+  - Added multivariate support to backtesting functions [\#111](https://github.com/unit8co/darts/pull/111)
+- Custom style for matplotlib plots [\#191](https://github.com/unit8co/darts/pull/191)
+- Option to pass a single string as value column name to the `TimeSeries.from_dataframe()` function. [\#114](https://github.com/unit8co/darts/pull/114)
+- sMAPE metric [\#129](https://github.com/unit8co/darts/pull/129)
+- `freq` parameter for TimeSeries generating functions [\#157](https://github.com/unit8co/darts/pull/157)
```

Comment:
Small nitpick: The `freq` parameter was already there, but now it is passed to the `TimeSeries` constructor. Not sure if we have to go into such details though.

Reply:
Oh that's right. We can probably leave that bullet point out then, what do you think ?

Reply:
I think so, I think it's more of a bugfix!